decidim-core/app/packs/src/decidim/editor.js

```diff
@@ -74,13 +89,30 @@ export default function createQuillEditor(container) {
         next(file);
       }
     }
+
+    const text = $(container).data("dragAndDropHelpText");
+    $(container).after(`<p class="help-text" style="margin-top:-1.5rem;">${text}</p>`);
   }
   const quill = new Quill(container, {
     modules: modules,
     formats: quillFormats,
     theme: "snow"
   });
 
+  if (addImage === false) {
+    quill.root.addEventListener("drop", (ev) => ev.preventDefault());
+    /* eslint-disable no-unused-vars */
+    quill.clipboard.addMatcher("IMG", (node, delta) => {
```

Comment:
Could you explain why we need to add these matches (`IMG` and `PICTURE`)?

There might be a valid reason for that but I'm not understanding it from this code only.

Reply:
This particular code is added to remove any image src or picture from the clipboard object. The place of this code should be in the clipboard_overrides but considering this fix is only for 2 versions (for 0.26 & 0.27)...

Reply:
If I comment out these lines, I am still not able to paste images or drop images to the editors at the participant side.

So it's still a bit to me unclear why we need to add these...

At admin side dropping works both in Chrome and Firefox with these lines commented out. Pasting works only in Firefox but that seems to be the case also at try.decidim.org, so there's no difference there ("invalid signature" error when the image is being uploaded).

To me it seems that these lines would be unnecessary. But if you can give me an exact replication instructions for the bug you're trying to fix with these lines, I can try it out again.

Reply:
@ahukkanen the solution comes from this comment: https://github.com/quilljs/quill/issues/1108#issuecomment-757547630, but apparently i cannot either replicate the issue anymore ( I will remove the code, as the quillFormat handler should do the trick)